main/job-stats.js

```diff
@@ -1,7 +1,23 @@
 'use strict'
 
+const { InfluxDB, Point } = require('@influxdata/influxdb-client')
 const Store = require('electron-store')
 
+const client = new InfluxDB({
+  url: 'https://eu-central-1-1.aws.cloud2.influxdata.com',
+  // station-anonymous-write
+  token: 'P9INe7-DkMB_oYSAXgw0IlBcqRc4bDi05jZFz0NZd8aApv5lieqBiN59M8cmHs6yNMRtly8_KD9RYxnyrnx8dA=='
+})
+const writeClient = client.getWriteApi(
+  'julian.gruber@protocol.ai',
+  'station',
+  'ns'
+)
```

Comment:
Where are these "magic string" values coming from, and what is their meaning? 

Is it a good idea to use your email address? Shouldn't we use some alias instead, e.g. `station@protocol.ai` or `station@filecoin.io`?

Reply:
I'm still figuring things out :) I agree we should use different credentials. The second value is the bucket name, the third I don't know yet (copied this over from their example). Will annotate these 👍